src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin watch-size($page-height) {
```

Comment:
It would be better to use abstract name for this mixin
'Cause you use it not only for watch container.

Reply:
I'm agree with you